backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/textfield/internal/BpkTextFieldImpl.kt

```diff
@@ -243,6 +252,14 @@ private fun TextFieldBox(
                     },
                 ).value,
             )
+        } else if (clearAction != null) {
+            BpkIcon(
```

Comment:
I don't see the similar to the start icon padding here – is that intentional?

Reply:
ooh, I will add an end padding here for symmetry.

Reply:
i don't think the padding is needed here as the text field already has padding as a whole which matches figma